### ISSUE TITLE
[Frontend] Resolve accessibility violation in Network Status Indicator

### ISSUE THREAD

**Issue by emdevelopa** [emdevelopa/Stellar_Payment_API#1378]
Description
This task involves UX enhancement for the Network Status Indicator module.
The goal is to resolve accessibility violation in Network Status Indicator to improve the platform's styling, user interactions, and overall accessibility.

Requirements and context

- Must be secure, tested, and documented
- Adhere to the Drips Wave design and UI/UX standards
- Ensure compatibility across desktop and mobile browsers
- Specifically focused on frontend UX enhancement and responsiveness

Suggested execution

Fork the repo and create a branch
git checkout -b feature/fe-resolve-accessibility-violation-in-network-status-indicator
Implement changes

Review existing component in Network Status Indicator
Apply changes: Resolve accessibility violation in Network Status Indicator
Use clean CSS or tailwind variables for styling
Maintain state transitions and visual feedback
Test and commit

Test mobile responsiveness and interactive states
Check accessibility (a11y) using standard audits
Include screenshots or gifs in the PR
Example commit message
feat(frontend): resolve accessibility violation in Network Status Indicator

Guidelines

Complexity: Medium (150 points)
Assignment required before starting
PR description must include: Closes #[issue_id]

**Comment by MarcusDavidG** [comment]
@MarcusDavidG has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> I can handle this issue and will like to work on it, Please assign thank you.


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/7cb3af88-af07-4b14-bf36-747a80669ec5) or assign @MarcusDavidG to this issue.

**Comment by LawalRahman** [comment]
@LawalRahman has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> Hi! I'd love to contribute to this project. I have experience building and maintaining production applications and enjoy working on well-scoped tasks. I focus on writing clean, maintainable code, communicating clearly, and delivering quality work on time. I'm excited about the opportunity to contribute and collaborate with the team.


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/7cb3af88-af07-4b14-bf36-747a80669ec5) or assign @LawalRahman to this issue.
